.kokoro/tests/run_tests.sh

```diff
@@ -121,6 +121,27 @@ set +e
 RTN=0
 ROOT=$(pwd)
 
+# Setup DRIFT region tag injector
+# (only run on *some* builds)
+if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
+    echo "=== Setting up DRIFT region tag injector ==="
+    # install PyYaml (used by the DRIFT region tag parsing system)
+
+    export REGION_TAG_PARSER_DIR="/tmp/region-tag-parser"
+    export PARSER_PATH="${REGION_TAG_PARSER_DIR}/wizard-py/cli.py"
+    export PIP_PATH=/tmp/pyyaml
```

Comment:
Why did you need PIP_PATH again?

Did you try:

```
pip install --user pyyaml -q 
```
?

Reply:
IIRC, I did (and it didn't work).

Reply:
What's the error message?

Reply:
If memory serves, it couldn't find the package (so probably something like `cannot find module 'yaml'`)

Reply:
I think it should work.
Can you try this locally and paste the error log?

Reply:
If I use that line and remove the `PYTHONPATH` modification:

```
Processing XUnit output file: /workspace/functions/env_vars/sponge_log.xml (saving output to /tmp/tmp.I0Hx5IhTZh)
Traceback (most recent call last):
  File "/tmp/region-tag-parser/wizard-py/cli.py", line 17, in <module>
    import analyze
  File "/tmp/region-tag-parser/wizard-py/analyze.py", line 19, in <module>
    import yaml_utils
  File "/tmp/region-tag-parser/wizard-py/yaml_utils.py", line 19, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

Reply:
Hmm maybe the package is being installed to a pip associated with a different Python version. Could you try something like `python3.7 -m pip install ...`?